src/components/Proposer/Proposer.story.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+	import type { Hst } from '@histoire/plugin-svelte';
+	import Proposer from './Proposer.svelte';
+	import { rep26 } from '../../mocks/data/assembly';
+	import { movingForwardPolitician } from '../../mocks/data/politician';
+	import { movingForwardParty } from '../../mocks/data/party';
+
+	export let Hst: Hst;
+</script>
+
+<Hst.Story title="Proposer">
+	<Proposer
```

Comment:
To make the story easier to understand, can you
- Split into 3 variants in grid layout ([see doc](https://histoire.dev/guide/svelte3/stories.html))
- Add control for the props ([see doc](https://histoire.dev/guide/svelte3/controls.html))